src/test/resources/application-local.yml

```diff
@@ -13,9 +13,9 @@ spring:
 
 no.nav.security.jwt:
   issuer.azuread:
-    discoveryurl: https://login.microsoftonline.com/navq.onmicrosoft.com/v2.0/.well-known/openid-configuration
-    accepted_audience: ${AZURE_CLIENT_ID}
-    cookie_name: azure_token
+    discoveryurl: http://localhost:${mock-oauth2-server.port}/azuread/.well-known/openid-configuration
```

Comment:
`local` brukes for lokal testing mot dev-microsoft, så dette blir ikke helt riktig?

Reply:
Det kjedelige med det er jo at konfigurasjonen ikke er komplett for å kunne starte applikasjonen. Er AZURE_CLIENT_ID for løkal kjøring så sensitiv at vi kkke kan sjekke den inn?

Reply:
Det er jeg enig i, men det var tidligere ønskelig att vi skulle ha integrasjon mot AAD.

Hvis man ønsker å kunne starte opp den uten eksterne integrasjoner uten å legge til en ekstra application-fil så hadde man kunnet løst det med eks
```yaml

---
# starte applikasjonen med env spring.profiles.active=local,uten-aad 
spring.config.activate.on-profile: uten-aad
no.nav.security.jwt:
  issuer.azuread:
    discoveryurl: ...
    accepted_audience: ...
    cookie_name: azure_token
    proxy_url: #Default satt, skal være null lokalt
```